compiler/rustc_target/src/spec/apple_base.rs

```diff
@@ -51,7 +53,7 @@ pub fn opts(os: &str) -> TargetOptions {
         // this environment variable too in recent versions.
         //
         // For some more info see the commentary on #47086
-        link_env: vec![("ZERO_AR_DATE".to_string(), "1".to_string())],
+        link_env: Cow::Borrowed(&[(Cow::Borrowed("ZERO_AR_DATE"), Cow::Borrowed("1"))]),
```

Comment:
```suggestion
        link_env: cfs!["ZERO_AR_DATE", "1"],
```

Reply:
This won't work as the `cvs!` macro is for Cow-Vec-Str not Cow-Vec-Tuple-Str.